smoke-test/src/test/java/org/opennms/smoketest/graph/GraphRestServiceIT.java

```diff
@@ -128,7 +132,65 @@ public void verifyGetContainer() {
                     .content("graphs[1].edges", Matchers.hasSize(0));
         });
     }
+    
+    @Test
+    public void verifySuggest() {
+    	createGraphMLAndWaitUntilDone();
+        given().params("s", "unknown")
+               .accept(ContentType.JSON)
+               .get("/search/suggestions/{namespace}/", "acme:regions")
+               .then().statusCode(204);
+        
+        Response response = given().params("s", "North Region")
+               .accept(ContentType.JSON)
+               .get("/search/suggestions/{namespace}/", "acme:regions");
+        LOG.info("Results from search:" + response.asString());
+        
+        response
+               .then().statusCode(200)
+               .contentType(ContentType.JSON)
+               .content("[0].context", Matchers.is("GenericVertex"))
```

Comment:
Let's also verify that the returned size of elements is correct.

Reply:
done.